src/utilities/frame/context.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import {createContext} from 'react';
 import {ToastPropsWithID, ToastID, ContextualSaveBarProps} from './types';
 
 export interface FrameContextType {
```

Comment:
Can we stop exposing the `FrameContextType` and instead use the `React.ContextType<typeof FrameContext>` inference instead to keep our API surface area small.